### PR TITLE
Style with styler

### PR DESCRIPTION
**Pull request by lorenzwalthert**
#296 to be merged into r-1.0.0 first.

### CODE REVIEW

File: R/serialized_tests.R [hunk 1]
```diff
@@ -160,23 +165,21 @@ style_empty <- function(text) {
...
 #' @describeIn test_transformer Transformations for indention based on operators
 style_op <- function(text) {
-
   transformers <- list(
     # transformer functions
     initialize = default_style_guide_attributes,
-    line_break = NULL,
-    space      = partial(indent_op, indent_by = 2),
-    token      = NULL,
+    line_break        = NULL,
```
Comment (krlmlr) [inline R/serialized_tests.R]: Are these manual changes? Need to align with `initialize` perhaps.

File: R/serialized_tests.R [hunk 2]
```diff
@@ -220,8 +223,9 @@ stop_insufficient_r_version <- function() {
 #' out manually.
 generate_test_samples <- function() {
   gen <- function(x) {
-    if (length(x) == 0) ""
-    else {
+    if (length(x) == 0) {
```
Comment (krlmlr) [inline R/serialized_tests.R]: Did we have a helper function for this? (Not relevant to this PR.)

File: R/serialized_tests.R (hunk 1 above)
Comment (lorenzwalthert) [reply R/serialized_tests.R]: Yes. Agree.

File: R/serialized_tests.R (hunk 2 above)
Comment (lorenzwalthert) [reply R/serialized_tests.R]: I just c/p your code from ##279.

Comment (lorenzwalthert) [reply R/serialized_tests.R]: So I suppose no.

Comment (krlmlr) [reply R/serialized_tests.R]: Oh, this is just the generator. Got it.

**Pull request merged**